tools/depends/pre-depends/m4-pre-depends/Makefile

```diff
@@ -1,19 +1,20 @@
 include ../../Makefile.include.in
```

Comment:
Fair enough. Ensures lots of env variables with werid values but it doesn't seem to be a problem.